src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -1876,7 +1867,7 @@ private File makeGesturesDirectory(String directory) {
     dir.mkdirs();
     if (!dir.isDirectory()) {
       // TODO: maybe create the directory ?
-      log.warn("Gestures directory {} doest not exist.", directory);
+      log.warn("Gestures directory {} doest not exist, will create it.", directory);
```

Comment:
This log message is not correct.  The line above dir.mkdirs() actually tries to create the directory.  This log message is in the event that you can't create the directory... one such example could be a file permissions issue.  please revert this log message as it's incorrect as to what is happening here.